src/pages/home.js

```diff
@@ -1,12 +1,55 @@
+// home.js
+
 import React from "react";
-import Stack from "react-bootstrap/Stack";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 import Card from "../components/card";
+import { FaChartLine } from "react-icons/fa6";
+import { SlLogin } from "react-icons/sl";
+import { FcFile } from "react-icons/fc";
 
 function Home() {
```

Comment:
While it is technically working, it does not look right when opening in the browser. The cards are too far apart from each other. Part of your responsibility as a developer is to make sure that the UI also looks good

<img width="1485" alt="image" src="https://github.com/daniel-august-mentorship/Demo-School/assets/6497134/d12c1fb3-572b-4a06-888c-6cf8f778bc6f">

Checkout the `Container` component to limit your contents in the middle of the screen so that even if the screen is too large its not too far apart from each other
https://react-bootstrap.netlify.app/docs/layout/grid#container